website/content/docs/plugins/plugin-development.mdx

```diff
@@ -26,7 +26,8 @@ a plugin is basic command-line skills and basic knowledge of the
 
 Your plugin implementation needs to satisfy the interface for the plugin
 type you want to build. You can find these definitions in the docs for the
-backend running the plugin.
+backend running the plugin. Note: Plugins should be prepared to handle multiple
```

Comment:
nit: remove `Note:` or use the arrow syntax to make the note box appear

```
~> Note: Plugins should ...
```

Reply:
Will do!